source/reference/minio-server/settings/kes.rst

```diff
@@ -36,10 +67,8 @@ Refer to your operating system's documentation for how to define an environment
 
    See the :minio-git:`KES wiki <kes/wiki/Configuration#policy-configuration>` for more complete documentation on KES access control.'
```

Comment:
rogue punctuation snuck in here at some point
```suggestion
   See the :minio-git:`KES wiki <kes/wiki/Configuration#policy-configuration>` for more complete documentation on KES access control.
```

Reply:
Where did that come from?
Gone now. Thanks!